src/components/common/Loading.tsx

```diff
@@ -0,0 +1,12 @@
+import { Container, Skeleton } from '@mui/material'
```

Comment:
I like the idea of moving the loading state out, but the idea of a skeleton is to have it look like the shape of the UI that you're loading. So having one loading state is a little weird if the UI that's rendered looks completely different.

So we could just do one big block as a skeleton to make it universal?

Reply:
Since all the pages have a title, breadscumbs and then a content. I will be adding 3 skeletons accodingly, wdyt ?

Reply:
Sounds good to me